docs/user-guide/creating-application/workflow/cd-pipeline.md

```diff
@@ -216,5 +219,10 @@ It terminates the old version and releases the new one.
 
 Devtron now supports attaching multiple deployment pipelines to a single build pipeline, in its workflow editor. This feature lets you deploy an image first to stage, run tests and then deploy the same image to production.
 
+Please follow the steps mentioned below to create sequential pipelines :
 
-![](../images/sequential-workflow.jpg)
\ No newline at end of file
+After creating CI/build pipeline, create a CD pipeline by clicking on the `+` sign on CI pipeline and configure the CD pipeline as per your requirements.
+To create CD pipeline for some other environment, click on `+` sign on CD pipeline.
```

Comment:
To add another CD Pipeline sequentially after previous one, again click on `+` sign on the last CD pipeline block.

Reply:
Edited